### ISSUE TITLE
6章：Batch Normalization サンプルコード

### ISSUE THREAD

**Issue by zenitani** [oreilly-japan/deep-learning-from-scratch#62]
p.189 6.3.2 節
ch06/batch_norm_test.py を実行させると
図6-18 が出てくるように読めるのですが、
実際には図6-19が現れます。
